hadoop-hdds/rocksdb-checkpoint-differ/src/test/java/org/apache/ozone/rocksdiff/TestRocksDBCheckpointDiffer.java

```diff
@@ -369,6 +355,55 @@ void diffAllSnapshots(RocksDBCheckpointDiffer differ) {
     }
   }
 
+  @ParameterizedTest
+  @ValueSource(ints = {0, 1, 2, 3, 4, 5})
+  public void testRemoveSnapshotFromDag(int snapshotIndex) throws Exception {
+    String clDirStr = "compaction-log";
+    String metadataDirStr = ".";
+    String sstDirStr = "compaction-sst-backup";
+    List<List<String>> expectedDiff = asList(
+        asList("000024", "000017", "000028", "000026", "000019", "000021"),
+        asList("000024", "000028", "000026", "000019", "000021"),
+        asList("000024", "000028", "000026", "000021"),
+        asList("000024", "000028", "000026"),
+        asList("000028", "000026"),
+        Collections.singletonList("000028"),
+        Collections.emptyList()
```

Comment:
Is this line (index 6) used anywhere?

Reply:
Actually index-0 is not needed. Removed index-0 line and added index-6 back.